code/oled_block/oled.py

```diff
@@ -0,0 +1,53 @@
+import Adafruit_GPIO.SPI as SPI
+import Adafruit_SSD1306
+import time
+
+from PIL import Image
+from PIL import ImageDraw
+from PIL import ImageFont
+
+oled_buffer_path = 'oled_buffer.txt' # Path to oled buffer
+line_width = 20 # Number of characters in one line
+sleep_time = 2 # Delay between displaying in seconds
+
+RST = 24 # Raspberry Pi RST pin, which we don't have to use xd
+
+disp = Adafruit_SSD1306.SSD1306_128_64(rst=RST) # 128x64 display with hardware I2C:
+
+disp.begin() # Initialize library.
+
+width = disp.width #some useful constants
+height = disp.height
+
+font = ImageFont.load_default() # Load default font.
+
+def get_message():
+    with open(oled_buffer_path, 'r') as file:
+        return file.read().replace('\n', '').replace('-','')
+
+def write_message(message):
+    disp.clear() #clear last message
+    disp.display()
+    y_pos = 0 #variable to store y axis position of line
+    image = Image.new('1', (width, height)) #our graphic with text
+
+    draw = ImageDraw.Draw(image) # Get drawing object to draw xd
+
+    split_string = [message[i:i+line_width] for i in xrange(0, len(message), line_width)]
+    for chunk in split_string[0:]:
```

Comment:
what is the purpose of this `split_string[0:]`, I don't see the point of that O.o

Reply:
Like why those '[]'? Don't know for now, copied that, gonna check later if we can get rid of them

Reply:
Consider the following example:
we've got a list with elements: `list = [0, 1, 2, 3, 4, 5]`, square bracket operator with colon inside slices the list (gets specified part of the list), in this case `list[1:3]` will return `[1, 2]`. So when slicing from element `0` (like in the like 41) we'll get same thing `list[0:] == [0, 1, 2, 3, 4, 5]`. That's why I don't see the point of that

Reply:
You Wright

Reply:
Resolved in commit: 855a8c5